core/modules.rs

```diff
@@ -436,15 +268,17 @@ impl ModuleNameMap {
 /// A collection of JS modules.
 #[derive(Default)]
 pub struct Modules {
-  info: HashMap<ModuleId, ModuleInfo>,
+  pub(crate) info: HashMap<ModuleId, ModuleInfo>,
   by_name: ModuleNameMap,
+  pub(crate) specifier_cache: HashMap<(String, String), ModuleSpecifier>,
 }
```

Comment:
I glossed over the creation of this struct in your previous review - but LGTM.

Reply:
Why pub(crate) here?

Reply:
`specifier_cache` is used to simplify module resolve callback - it is no longer needed to create closures (which were wrong anyway...) and pass pointers to them. Everything is now encapsulated in `Isolate::module_resolve_cb` method